back/lib/tasks/single_use/20250326_migrate_forms_separate_title_description_pages.rake

```diff
@@ -0,0 +1,284 @@
+namespace :migrate_custom_forms do
+  desc 'Migrate forms to have separate pages for title and body.'
+  task :separate_title_body_pages, [] => [:environment] do
+    reporter = ScriptReporter.new
+    Tenant.safe_switch_each do |tenant|
+      {
+        'ideation_page1' => 'title_page',
+        'ideation_page2' => 'uploads_page',
+        'ideation_page3' => 'details_page'
+      }.each do |code_from, code_to|
+        CustomField.where(input_type: 'page', code: code_from).update_all(code: code_to)
+      end
```

Comment:
Notice these all have a 'key' of `nil` by default whereas native_survey has nil code and the page has a key. Wonder if we need to give these a key too? But I guess if it works as it is then maybe we should leave in case it breaks something else!

Reply:
I think we've had the conversation of "code" vs. "key" a few times (but the only documentation I could find back is my own 😅 https://www.notion.so/govocal/Custom-fields-a1342b71b1f64a8eaefd796bbc8144f1?pvs=4#88dc3d8517f843dc82b6dd025352ee8d).

In short, I think to remember that "code" is used to find back fields that have a unique special role in the form (like the `title_multiloc` field), whereas the "key" is would only be used for the custom_field_values. I would therefore argue that the "code" is the right attribute in these cases, and that setting a "key" for pages wouldn't be very meaningful.